bittensor/_keyfile/keyfile_impl.py

```diff
@@ -227,8 +259,17 @@ def decrypt_keyfile_data(keyfile_data: bytes, password: str = None, coldkey_name
         password = getpass.getpass("Enter password to unlock key: ") if password is None else password
         console = bittensor.__console__;             
         with console.status(":key: Decrypting key..."):
+            
+            # NaCl SecretBox decrypt.
+            if keyfile_data_is_encrypted_nacl( keyfile_data ):
+                password = bytes(password, 'utf-8')
+                kdf = pwhash.argon2i.kdf
+                key = kdf(secret.SecretBox.KEY_SIZE, password, NACL_SALT, opslimit=pwhash.argon2i.OPSLIMIT_SENSITIVE, memlimit=pwhash.argon2i.MEMLIMIT_SENSITIVE)
+                box = secret.SecretBox(key)
+                decrypted_keyfile_data = box.decrypt(keyfile_data[len('$NACL'):])
```

Comment:
another usage of the value where we could use the variable/constant with '$NACL'